src/OrderProperties/BqOrderPropertiesManager.php

```diff
@@ -59,18 +68,21 @@ private function createProperty(string $value, string $propertyIdentifier, strin
         $sessionProperties = $this->bqRestManagerV1->get('session')->default_properties;
         $matchingProperties = array_filter($sessionProperties, fn($p) => $p->identifier === $propertyIdentifier);
         if (count($matchingProperties) === 1) {
-            $propertyType = array_pop($matchingProperties)->property_type;
+            $propertyType = PropertyTypes::from(array_pop($matchingProperties)->property_type);
         } else {
             throw new BqRequestException("no matching property found in session");
         }
+        if (!$propertyType->isStringProperty()) {
+            throw new BqRequestException("property isn't a string property (allowed are: Single- and MultiLineText)");
```

Comment:
Bitte eine RuntimeException daraus machen. Es ist ja nicht zu erwarten, dass dies passiert und das hat ja auch nicht wirklich etwas mit dem BqRequest zu tun.  

Die message bitte zu "currently, only single- and multi-line-text-properties are tested"